### PR TITLE
Fix outdated unattended install command and syntax

### PR DESCRIPTION
**Pull request by medv**
Finishes https://github.com/AstroNvim/astronvim.github.io/pull/63

### CODE REVIEW

File: docs/Recipes/unattended_install.md [hunk 1]
```diff
@@ -14,4 +14,4 @@ Full Steps:
 
 1. `git clone --depth 1 https://github.com/AstroNvim/AstroNvim ~/.config/nvim`
 2. (Optional) `git clone`/copy over user configuration to `~/.config/nvim/user`
-3. `nvim --headless -c 'autocmd User LazyDone quitall'
+3. `nvim --headless -c 'quitall'`
```
Comment (mehalter) [inline docs/Recipes/unattended_install.md:17]:
```suggestion
3. `nvim --headless +q`
```

**Review by mehalter — CHANGES_REQUESTED** [review]
Thanks for catching this! I did some testing and it does seem like we don't want to have the autocmd. In this case qw can also do `+q` instead of the full `-c 'quitall'`

**Comment by medv** [comment]
This answered a couple questions actually, thanks!

**Review by mehalter — APPROVED** [review]
Looks great! Thanks :D

**Pull request merged**